_config.yml

```diff
@@ -801,15 +801,10 @@ rating:
 
 # Show number of visitors to each article.
 # You can visit https://leancloud.cn to get AppID and AppKey.
+# Dependencies: https://github.com/LEAFERx/hexo-leancloud-counter/
+# Checkout https://github.com/LEAFERx/hexo-leancloud-counter/ for how to setup.
 leancloud_visitors:
   enable: false
```

Comment:
How about using only one line here:
```yml
leancloud_visitors: false
```

Reply:
And link on NexT website.

@stevenjoezhang are we will move from `docs` to site all excess docs? P.S. Need to set up Slack, that's impossible to work: it is not clear whom and what to do.

Reply:
@ivan-nginx I'm not sure where to put the `DEPRECATED` warning would be better.
Should let users know about `leancloud-counter-security` and `exturl.js` (https://github.com/theme-next/hexo-theme-next/blob/master/scripts/tags/exturl.js, will be removed in next release)

Reply:
Like I said, I may implement hot post feature and config will be like
```yml
leancloud_visitors:
  enable: false
  hotpost: false
```

Now all links has point to NexT.